### PR TITLE
Orm

### PR DESCRIPTION
**Pull request by mayppong**
Here's a start for ORM support added to the project. Aside from cleaner code, we also now have promise objects to DB calls. Hurray.

### CODE REVIEW

File: config/database.json [hunk 1]
```diff
@@ -8,14 +8,14 @@
...
-    "password": null,
+    "password": "password",
     "database": "gyro_test",
     "host": "127.0.0.1",
     "dialect": "mysql"
   },
   "production": {
     "username": "root",
-    "password": null,
+    "password": "password",
```
Comment (whiterook6) [inline config/database.json]: what's this about?

File: lib/game.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ Game = {
 
 	create_player: function(socket){
 		Logger.debug('Game.create_player');
-		Game.players.push(new Player(socket));
+		Game.players.push(new PlayerController(socket));
```
Comment (whiterook6) [inline lib/game.js:19]: If this has to be called a Player Controller, can you change the Game.players list to be Game.playerControllers?

File: models/player.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+		},
+
+		initials: {
+			type: Sequelize.CHAR,
+			allowNull: false,
+			validate: {
+				len: 3
+			},
+			set: function(value) {
+				this.setDataValue('initials', value.trim().toUpperCase().substring(0, 3));
```
Comment (whiterook6) [inline models/player.js:16]: nice

File: config/database.json (hunk 1 above)
Comment (mayppong) [inline config/database.json]: I guess we don't have to do one for the production environment. I really just want to do it for the test environment. I will change this back to null.

File: lib/game.js (hunk 2 above)
Comment (mayppong) [inline lib/game.js:19]: This is just temporary. Eventually we'll still want to store a list of player, not the player controller object. That'd be weird.

**Pull request merged**